html/styles.css

```diff
@@ -0,0 +1,378 @@
+}
+
+.page-header .checkout-balance {
+  height: 30px;
+  color: white;
+  position: absolute;
+  top: 20px;
+  right: 100px;
+}
+
+.page-header .checkout-balance .cart-icon {
+  width: 20px;
+  margin-right: 2px;
+}
+
+.page-header .checkout-balance .dollar {
+  font-weight: 600;
+  font-size: 8pt;
+}
+
+.page-header .main-title {
+  margin-left: 100px;
+  margin-top: 0;
+  font-family: sans-serif;
+  font-size: 14pt;
+  color: #81B841;
+}
+
+.page-header .button-checkout {
+  background-color: #81B841;
+  border: none;
+  color: white;
+  padding: 5px 20px;
+  font-size: 7pt;
+}
+
+.page-header .button-checkout {
+  position: absolute;
+  right: 95px;
+  top: 50px;
+  text-transform: capitalize;
+  text-decoration: none; 
+}
+
+.page-header .button-checkout:hover {
+  opacity: 0.9;
+}
+
+.topnav {
+  background-color: #e0e0e0;
+  padding: 20px 10%;
+  font-size: 10pt;
+  
+  font-weight: bold;
+  word-spacing: 5px;
+}
+
+.navigation-link {
+  text-decoration: none;
+  color: #134350;
+  font-weight: 800;
+}
+
+main {
+  width: 80%;
+  margin: auto;
+  padding-bottom: 50px;
+}
+
+.introduction {
+  display: flex;
+  align-items: flex-start;
+  margin: 20px 0;
+}
+
+.introduction .intro-image {
+  width: 35%;
+  margin-right: 40px;
+}
+
+.introduction .intro-text .title {
+  color: #134350;
+  font-family: 'Times New Roman', Times, serif;
+}
+
+.introduction .intro-text .intro-paragraph {
+  font-size: 12pt;
+  color: #444;
+  line-height: 25px;
+}
+
+.dog-grid {
+  display: grid;
+  grid-template-columns: auto auto auto auto;
+  grid-gap: 20px;
+  justify-content: center;
+  margin-bottom: 30px;
+}
+
+@media only screen and (max-width: 650px) {
+  .dog-grid {
+    grid-template-columns: auto auto; 
+  }
+}
+
+
+
+.dog-grid .dog-card {
+  border: solid 2px gray;
+  overflow: hidden;
+}
+
+
+.dog-grid .dog-card .dog-picture {
+  width: 100%;
+}
+
+.dog-grid .dog-card .dog-name {
+  color:  #134350;
+  font-family: 'Times New Roman', Times, serif; 
+  font-size: 1.3rem;
+  margin: 0 10px;
+}
+
+.dog-grid .dog-card .dog-cost {
+  margin: 5px 10px;
+  font-size: 0.9rem;
+}
+
+.dog-grid .dog-card .dog-description {
+  margin: 20px 10px;
+  font-size: 0.95rem;
+  line-height: 1.2rem;
+}
+
+.dog-grid .dog-card .dog-link {
+  display: block;
+  text-align: center;
+  margin: 10px;
+  padding: 8px;
+  font-weight: bold;
+  background-color:  #81B841;
+  color: white;
+}
+
+.dog-grid .dog-card .dog-link:hover {
+  opacity: 0.9;
+} 
+
+.see-more-dogs {
+  display: block;
+  margin: 20px auto 80px auto;
+  width: 180px;
+  background-color: #eee;
+  color: #134350;
+  padding: 10px 20px;
+  text-align: center;
+  text-transform: capitalize;
+  font-weight: bold;
+}
+
+.blog-title, 
+.title-all-dogs, 
+.page-title {
+  color: #134350;
+  font-family: 'Times New Roman', Times, serif;
+  font-size: 1.4rem;
+}
+
+.blog-story {
+  display: flex;
+  flex-direction: row;
+  margin: 20px 0;
+  align-items: center;
+}
+
+.blog-story .blog-image {
+  width: 350px;
```

Comment:
Would also look to make your blog images larger as well to better match the spec. I would suggest using a responsive width size instead to ensure the image looks right on all screen sizes.